lisa/sut_orchestrator/libvirt/schema.py

```diff
@@ -68,6 +74,9 @@ class BaseLibvirtNodeSchema:
     disk_img_resize_gib: Optional[int] = None
     # Configuration options for cloud-init.
     cloud_init: Optional[CloudInitSchema] = None
+
+    # Configuration options for ignition.
+    ignition: Optional[IgnitionSchema] = None
```

Comment:
Add `__post_init__` to validate the schema, make sure cloud_init and ignition not set together.

Reply:
Refer to `lisa/schema.py`. The class `Variable`, `Platform` and `ConnectionInfo` raise exception in `__post_init__`. But this is an extension runbook, maybe the behavior is different. Please help checking if log has the exception dumpped, but not raised out. I will take a look if it is.